test/quantization/core/test_quantized_op.py

```diff
@@ -253,7 +255,12 @@ def test_qrelu(self, X):
                 }
             }
         ]
-        self._test_activation_function(X, 'relu', relu_test_configs)
+        devices = ["cpu", "cuda"] if TEST_CUDA else ["cpu"]
```

Comment:
this reminds me, i forgot to land https://github.com/pytorch/pytorch/pull/79607

Reply:
it's currently landing, so i think you can change it after i land

Reply:
OK